draw/src/main/res/values/colors.xml

```diff
@@ -10,4 +10,14 @@
     <color name="color_blue">#00B0FF</color>
```

Comment:
Shouldn't this file be part of to the library?

Reply:
This color file is part of the "draw" package, but there is no need of the Color-array.  I forgot to remove it.